packages/lwt/lwt.4.2.1/opam

```diff
@@ -0,0 +1,58 @@
+opam-version: "2.0"
+
+synopsis: "Promises and event-driven I/O"
+
+license: "MIT"
+homepage: "https://github.com/ocsigen/lwt"
+doc: "https://ocsigen.org/lwt/manual/"
+bug-reports: "https://github.com/ocsigen/lwt/issues"
+
+authors: [
+  "Jérôme Vouillon"
+  "Jérémie Dimino"
+]
+maintainer: [
+  "Anton Bachin <antonbachin@yahoo.com>"
+  "Mauricio Fernandez <mfp@acm.org>"
+  "Simon Cruanes <simon.cruanes.2007@m4x.org>"
+]
+dev-repo: "git+https://github.com/ocsigen/lwt.git"
+
+depends: [
+  "cppo" {build & >= "1.1.0"}
+  "dune" {build}
+  "mmap"  # mmap is needed as long as Lwt supports OCaml < 4.06.0.
+  "ocaml" {>= "4.02.0"}
+  "result" # result is needed as long as Lwt supports OCaml 4.02.
+  "seq" # seq is needed as long as Lwt supports OCaml < 4.07.0.
+
+  "bisect_ppx" {dev & >= "1.3.0"}
+  "ocamlfind" {dev & >= "1.7.3-1"}
```

Comment:
This is not blocker at all but I'm curious why there is still a dependency on ocamlfind

Reply:
The PPX expect tests [use](https://github.com/ocsigen/lwt/blob/6213e180686f7776781e9a50305792710df8d915/test/ppx_expect/main.ml#L54) it as an easy way of invoking the compiler.